test/Microsoft.Azure.Devices.Edge.Test.Common/certs/RootCertificateAuthority.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common.Certs
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public class RootCertificateAuthority
+    {
+        readonly string scriptPath;
+
+        public static async Task<RootCertificateAuthority> CreateAsync(string certificatePath, string keyPath, string password, string scriptPath, CancellationToken token)
+        {
+            await Platform.InstallRootCertificateAsync(certificatePath, keyPath, password, scriptPath, token);
+            return new RootCertificateAuthority(
+                certificatePath,
+                keyPath,
+                scriptPath);
+        }
+
+        RootCertificateAuthority(string certificatePath, string keyPath, string scriptPath)
+        {
+            this.scriptPath = scriptPath;
+        }
+
+        public Task<EdgeCertificateAuthority> CreateEdgeCertificateAuthorityAsync(string deviceId, CancellationToken token) =>
```

Comment:
what if adding scriptPath as parameter for this method and make it as static? then this class can be state as well?

Reply:
`RootCertificateAuthority` only exists to create an `EdgeCertificateAuthority`. I planned to collapse those two into one, but then decided against it. I'll take another look at doing that.

Reply:
Done. `RootCertificateAuthority` is gone; `EdgeCertificateAuthority` has been renamed to just `CertificateAuthority`.